metadata.json

```diff
@@ -59,37 +59,43 @@
     {
       "operatingsystem": "RedHat",
       "operatingsystemrelease": [
-        "7"
+        "7",
+	"8"
```

Comment:
please align the new entries so they have the same indentation.

Reply:
Hum, sorry about that, vim preferred tabs it seems